playbooks/adhoc/uninstall.yml

```diff
@@ -182,19 +185,35 @@
     - registry\.qe\.openshift\.com/.*
     - registry\.access\..*redhat\.com/rhel7/etcd
     - docker.io/openshift
-    when: openshift_uninstall_images | default(True) | bool
 
   - shell: "docker rmi -f {{ item.stdout_lines | join(' ') }}"
+    when: openshift_uninstall_images | default(True) | bool and openshift_remove_all | default(True) | bool
     changed_when: False
     failed_when: False
     with_items: "{{ images_to_delete.results }}"
-    when: openshift_uninstall_images | default(True) | bool
 
   - name: Remove sdn drop files
+    when: openshift_remove_all | default(True) | bool
     file:
       path: /run/openshift-sdn
       state: absent
 
+  - name: Remove generated files
+    file: path={{ item }} state=absent
+    with_fileglob:
+    - /etc/origin/master/*
+    - /etc/origin/node/*
```

Comment:
I think you should be safe to just remove /etc/origin/master and /etc/origin/node, without issues. The installer at least should create those if absent.